src/daemon/bundle.rs

```diff
@@ -5,11 +5,15 @@ use anyhow::Context;
 use async_compression::futures::bufread::ZstdDecoder;
 use cid::Cid;
 use fvm_ipld_blockstore::Blockstore;
+use static_assertions::const_assert_eq;
 
 pub async fn load_actor_bundles(db: &impl Blockstore) -> anyhow::Result<Vec<Cid>> {
     const ERROR_MESSAGE: &str = "Actor bundles assets are not properly downloaded, make sure git-lfs is installed and run `git lfs pull` again. See <https://github.com/git-lfs/git-lfs/blob/main/INSTALLING.md>";
+
     const ACTOR_BUNDLES_CAR_ZST: &[u8] = include_bytes!("../../assets/actor_bundles.car.zst");
-    assert!(ACTOR_BUNDLES_CAR_ZST.len() > 1024 * 1024, "{ERROR_MESSAGE}");
+    // Check bundle size at compile time, see `ERROR_MESSAGE` for details.
+    // Note: make sure the bundle size is updated with the bundle file.
+    const_assert_eq!(ACTOR_BUNDLES_CAR_ZST.len(), 2438387);
```

Comment:
For the sake of completeness, it'd be helpful to provide instructions on how to regenerate the bundle. This way, a potential network upgrade implementor would be less frustrated by this sudden obstacle.

Reply:
I'm working on a `forest-tool` subcommand that regenerates the bundle, and will update instructions in STATE_MIGRATION.md altogether